rust/libnewsboat/src/utils.rs

```diff
@@ -595,6 +595,17 @@ mod tests {
         );
     }
 
+    #[test]
+    fn t_quote_url_for_cmdline() {
+        assert_eq!(quote_url_for_cmdline("".to_string()), "''");
+        assert_eq!(quote_url_for_cmdline("'".to_string()), "'%27'");
+        assert_eq!(quote_url_for_cmdline("'one'two'".to_string()), "'%27one%27two%27'");
```

Comment:
This line is too long, as `cargo fmt` [points out](https://travis-ci.org/newsboat/newsboat/jobs/490908516#L475). Please run `cargo fmt` and commit the result.

Reply:
Done